.github/workflows/commit.yaml

```diff
@@ -47,9 +47,10 @@ jobs:
         with:
           path: |
             ~/.cache/go-build
+            ~/.cache/golangci-lint
             ~/go/pkg/mod
             ~/go/bin
-          key: check-${{ runner.os }}-go-${{ matrix.go-version }}-${{ hashFiles('**/go.sum', 'Makefile') }}
+          key: cache-${{ runner.os }}-go-${{ matrix.go-version }}-${{ hashFiles('**/go.sum', 'Makefile') }}
```

Comment:
sorry the new cache has been saved successfully without key change: https://github.com/tetratelabs/wazero/runs/8242041868?check_suite_focus=true

could you revert this?